package.json

```diff
@@ -12,8 +12,8 @@
         "test:unit:watch": "lwc-jest --watch",
         "test:unit:debug": "lwc-jest --debug",
         "test:unit:coverage": "lwc-jest --coverage",
-        "prettier": "prettier --write '**/*.{cmp,component,css,html,js,json,md,page,yaml,yml}'",
-        "prettier:verify": "prettier --list-different '**/*.{html,js,json,yaml,yml,md,cmp,page,component}'",
+        "prettier": "prettier --write '**/*.{cmp,component,css,html,js,json,md,page,yaml,yml,cls}'",
```

Comment:
Triggers are missing. Also `lint-staged` is not updated.

Reply:
Also alphabetical order would be great. ;-)

Reply:
@muenzpraeger ready.